apps/dh/api-dh/source/DataHub.WebApi/GraphQL/Mutation/SettlementReportMutation.cs

```diff
@@ -22,8 +23,13 @@ public partial class Mutation
 {
     public async Task<bool> RequestSettlementReportAsync(
         RequestSettlementReportInput requestSettlementReportInput,
+        [Service] IMarketParticipantClient_V1 marketPartClient,
         [Service] ISettlementReportsClient client)
     {
+        var requestAsActor = string.IsNullOrEmpty(requestSettlementReportInput.RequestAsActorId)
```

Comment:
Instead of `string.IsNull..` you could use `Guid.TryParse(..)`

Reply:
Also, indentation seems of :D

Reply:
Fixed